eng/templates/jobs/darc-and-api-tests.yml

```diff
@@ -0,0 +1,77 @@
+jobs:
+- job: Darc_and_API_tests
+  displayName: Darc and API tests
+
+  pool:
+      name: NetCore1ESPool-Internal
+      demands: ImageOverride -equals 1es-windows-2019
+
+  steps:
+  - powershell: |
+      try {
+        $response = Invoke-WebRequest https://maestro.dot.net/api/builds/254867?api-version=2020-02-20
```

Comment:
This must be env dependent

Reply:
Also not great to include a specific build. If it gets deleted, this test will fail because we get 404.

I'd rather do some generic call that has no specific input and will work cross envs.

Reply:
Yeah I need to fix the env, my bad. But I think it's ok to use a specific build. If we get anything else other than a 401 we should fail. If we get a 404, we got passed auth which should fail the test

Reply:
Fair point with the 404 but I'd still use something like `https://maestro.dot.net/api/channels?api-version=2020-02-20`